bin/oref0-setup.sh

```diff
@@ -377,14 +403,14 @@ if [[ "$ttyport" =~ "spi" ]]; then
 fi
 
 cd $directory || die "Can't cd $directory"
-if [[ -z "$ttyport" ]]; then
+if [[ -z "$using_mmeowlink" ]]; then
     openaps device add pump medtronic $serial || die "Can't add pump"
     # carelinks can't listen for silence or mmtune, so just do a preflight check instead
     openaps alias add wait-for-silence 'report invoke monitor/temp_basal.json'
     openaps alias add wait-for-long-silence 'report invoke monitor/temp_basal.json'
     openaps alias add mmtune 'report invoke monitor/temp_basal.json'
 else
-    openaps device add pump mmeowlink subg_rfspy $ttyport $serial || die "Can't add pump"
+    openaps device add pump mmeowlink subg_rfspy $ttyport $serial $radio_locale || die "Can't add pump"
```

Comment:
We also need to update oref0-setup to install the version of openaps with radio_locale support before this will work, right?